src/redisraft.h

```diff
@@ -965,4 +995,16 @@ void ClientStateFree(RedisRaftCtx *rr, unsigned long long client_id);
 void ClientStateReset(ClientState *client_state);
 void MultiStateReset(MultiState *multi_state);
 
+/* blocked.c */
+BlockedCommand *allocBlockedCommand(const char *cmd_name, raft_index_t idx, raft_session_t session, const char *data, size_t data_len, RaftReq *req, RedisModuleCallReply *reply);
+void addBlockedCommand(BlockedCommand *bc);
+void freeBlockedCommand(BlockedCommand *bc);
+void deleteBlockedCommand(raft_index_t idx);
+BlockedCommand *getBlockedCommand(raft_index_t idx);
+void blockedCommandsSave(RedisModuleIO *rdb);
+void blockedCommandsLoad(RedisModuleIO *rdb);
+void clearAllBlockCommands();
+int RaftRedisExtractBlockingTimeout(RedisModuleCtx *ctx, RaftRedisCommandArray *cmds, long long *timeout);
+void RaftRedisReplaceBlockingTimeout(RaftRedisCommandArray *cmds);
```

Comment:
Not really sure why we need RaftRedis prefix for these two commands.

Reply:
ok